packages/console/src/pages/SignInExperience/PageContent/Branding/UpsellNotice/index.module.scss

```diff
@@ -0,0 +1,53 @@
+@use '@/scss/underscore' as _;
+
+.inlineNotification {
+  padding: _.unit(3) _.unit(4);
+  font: var(--font-body-2);
+  display: flex;
+  border-radius: 8px;
+  align-items: center;
+  gap: _.unit(3);
+  margin-bottom: _.unit(4);
+
+  &.shadow {
+    border: 1px solid var(--color-border);
+    box-shadow: var(--shadow-1);
+  }
+
+  .icon {
+    flex-shrink: 0;
+    width: 20px;
+    height: 20px;
+  }
+
+  .content {
+    flex: 1;
+    overflow: hidden;
+    overflow-wrap: break-word;
+
+    // Cancel the margin from MDX generated paragraphs
+    &:has(> p) {
+      margin: _.unit(-4) 0;
+    }
+  }
+
+  &.info {
+    background: var(--color-info-container);
+
+    .icon {
+      color: var(--color-on-info-container);
+    }
+  }
+
+  &.alert {
+    background: var(--color-alert-container);
+  }
+
+  &.success {
+    background: var(--color-success-container);
+  }
+
+  &.error {
+    background: var(--color-error-container);
+  }
```

Comment:
It seems these CSS classes are not in-use in this component. Are they copied from another component? If yes, is it possible to re-use the existing component instead of creating a new one?